ftd/src/interpreter/main.rs

```diff
@@ -92,6 +93,28 @@ impl InterpreterState {
         }
     }
 
+    /// [InterpreterState] with stub functions replaces with fastn_core::Package dependant
+    /// functions.
+    /// You pass real functions using the `builtins` parameter.
+    #[tracing::instrument(skip(builtins))]
+    fn new_with_expanded_builtins(
+        id: String,
+        builtins: impl IntoIterator<Item = (String, fastn_resolved::Definition)>,
```

Comment:
I usually prefer `builtins: &[(String, fastn_resolved::Definition)]`. While yours is more generic, this is slightly less to type (and relies of the fact that it will be a vector or array to begin with, not a general purpose iterator).

Reply:
done in: https://github.com/fastn-stack/fastn/pull/2057/commits/897fc62b9098dcf1ef6cca4629f8fd5679501c7b